### DISCUSSION TITLE
I have fixed the saving problem of extra armors and dyes in TShock for 1.4.4.x ssc.

### DISCUSSION THREAD

**Discussion by Leader-txt** [Pryaxis/TShock#2741] [Show and tell]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Leader-txt** [comment]
Project:[here](https://github.com/Leader-txt/FixSSC_Temp)

**Comment by punchready** [comment]
Unfortunately this is partially unnecessary (saving the loadout data in a separate table) and broken because it is subject to the same issue that the tShock implementation is currently working on.

**Comment by Leader-txt** [comment]
> 

Sounds good,And it's a special type of urging the update. : )
